package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "auth0": "^4.2.0",
+    "dompurify": "^3.0.6",
```

Comment:
Love this, happy you were able to find it. do I need to install dompurify locally to get this to work? or will it just be included since its in the package now?